src/sentry/event_manager.py

```diff
@@ -786,11 +798,16 @@ def _auto_update_grouping(project: Project) -> None:
         )
 
 
-@metrics.wraps("event_manager.background_grouping")
 def _calculate_background_grouping(
```

Comment:
FYI I think this is dead code. I can't see that metric used in DD.

Reply:
I've added a TODO